app/todo/service.py

```diff
@@ -0,0 +1,37 @@
+import abc
+
+from app.errors.excpetion import NotFoundException
+from app.todo.repository import BaseTodoRepository
+from app.todo.models import TodoInfo
+
+class BaseTodoService(abc.ABC):
+    def __init__(self, repository: BaseTodoRepository) -> None:
+        self.todo_repository = repository
+
+    @abc.abstractmethod
+    async def getTodoInfoList(self, user_id: int) -> list[TodoInfo]:
+        raise NotImplementedError
+    
+    @abc.abstractmethod
+    async def insetTodoInfo(self, user_id: int, context: str) -> TodoInfo:
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    async def deleteTodoInfo(self, user_id:int, todo_id: int) -> int:
+        raise NotImplementedError

```

Comment:
method를 snake_case로 통일하면 좋을 것 같습니다! (p4)
```suggestion
class BaseTodoService(abc.ABC):
    def __init__(self, repository: BaseTodoRepository) -> None:
        self.todo_repository = repository

    @abc.abstractmethod
    async def fetch_todo(self, user_id: int) -> list[TodoInfo]:
        raise NotImplementedError
    
    @abc.abstractmethod
    async def add_todo(self, user_id: int, context: str) -> TodoInfo:
        raise NotImplementedError

    @abc.abstractmethod
    async def remove_todo(self, user_id:int, todo_id: int) -> int:
        raise NotImplementedError

```